experimental/FTheoryTools/src/G4Fluxes/properties.jl

```diff
@@ -52,9 +52,67 @@ true
   # now execute elementary checks of the quantization condition
   for i in 1:length(c_ds)
     for j in i:length(c_ds)
-      numb = integrate(cohomology_class(ambient_space(m), twist_g4 * cy * c_ds[i] * c_ds[j]); check = false)
+      numb = integrate(cohomology_class(ambient_space(m), twist_g4 * c_ds[i] * c_ds[j] * cy); check = false)
       !is_integer(numb) && return false
     end
   end
   return true
 end
+
+
+@doc raw"""
+    passes_verticality_checks(gf::G4Flux)
+
+G4-fluxes are subject to verticality conditions described in [Lin16](@cite).
+It is hard to verify that these condition are met. However,
+we can execute a number of simple consistency checks, by
+verifying that $\int_{Y}{G_4 \wedge [D_1] \wedge [zero section]} = 0$ and $\int_{Y}{G_4 \wedge [D_1] \wedge [D_2]} = 0$
+for all toric base divisors $D_1$ and $D_2$. If all of these
+simple consistency checks are met, this method will return
+`true` and otherwise `false`
+
+```jldoctest
+julia> qsm_model = literature_model(arxiv_id = "1903.00009", model_parameters = Dict("k" => 4))
+Hypersurface model over a concrete base
+
+julia> divs = torusinvariant_prime_divisors(ambient_space(qsm_model));
+
+julia> e1 = cohomology_class(divs[35]);e2 = cohomology_class(divs[32]);e4 = cohomology_class(divs[34]);u = cohomology_class(divs[33]);v = cohomology_class(divs[30]);pb_Kbar = cohomology_class(sum([divs[k] for k in 1:29]));
+
+julia> g4_class = (-3) // kbar3(qsm_model) * (5 * e1 * e4 + pb_Kbar * (-3 * e1 - 2 * e2 - 6 * e4 + pb_Kbar - 4 * u + v))
+Cohomology class on a normal toric variety given by 1//2*x5*x32 + x5*x33 + 3//2*x5*x34 + x5*x35 + 1//2*x5*x36 + 1//2*x6*x32 + x6*x33 + 3//2*x6*x34 + x6*x35 + 1//2*x6*x36 - x7*x12 + 1//2*x7*x32 + x7*x33 + 3//2*x7*x34 + x7*x35 + 1//2*x7*x36 - 4*x8*x23 - 2*x10*x23 + 2*x15*x26 + 1//2*x15*x32 + x15*x33 + 3//2*x15*x34 + x15*x35 + 1//2*x15*x36 + x16^2 + 1//2*x16*x32 + x16*x33 + 3//2*x16*x34 + x16*x35 + 1//2*x16*x36 + 11*x17*x24 + x17*x32 + 2*x17*x33 + 3*x17*x34 + 2*x17*x35 + x17*x36 + 5*x18^2 - 7//4*x18*x25 + x18*x32 + 2*x18*x33 + 3*x18*x34 + 2*x18*x35 + x18*x36 + 3//2*x19*x32 + 3*x19*x33 + 9//2*x19*x34 + 3*x19*x35 + 3//2*x19*x36 - 4*x20^2 - 44//3*x20*x21 + 7//2*x20*x25 + x20*x32 + 2*x20*x33 + 3*x20*x34 + 2*x20*x35 + x20*x36 - 11//6*x21^2 + 77//6*x21*x24 + x21*x32 + 2*x21*x33 + 3*x21*x34 + 2*x21*x35 + x21*x36 - 31//6*x22^2 - 5//6*x22*x23 - 97//12*x22*x25 - 2*x23^2 + 4*x24^2 + 17//6*x24*x27 + 1//2*x24*x32 + x24*x33 + 3//2*x24*x34 + x24*x35 + 1//2*x24*x36 - 7//4*x25^2 + 1//2*x25*x32 + x25*x33 + 3//2*x25*x34 + x25*x35 + 1//2*x25*x36 + 1//3*x26*x27 - 5//6*x27^2 + 1//2*x27*x32 + x27*x33 + 3//2*x27*x34 + x27*x35 + 1//2*x27*x36 - 1//2*x28^2 + 7//6*x29^2 - 5//2*x35*x36
```

Comment:
Maybe the output for this one could be suppressed? Again, I am thinking about the readability of the docs?

Reply:
You're right I will make the docs more readable